Library/Homebrew/utils/collection.rb

```diff
@@ -0,0 +1,31 @@
+# typed: true
+# frozen_string_literal: true
+
+module Utils
+  module Collection
```

Comment:
Is expectation that this will be used for Casks too?

Obviously no rush yet but: good to have tests for this.